docs/components-upgrade.md

```diff
@@ -82,3 +84,61 @@ If something goes wrong with the upgrade, or if tests are not successful after u
 ### Hotfix "in-place" upgrades
 
 This process will restart tomcat on all the repository nodes and there is no guarantee one node is stopped only after the others have restarted. As a consequence a short service outage needs to be scheduled
+
+## Specifing a different component repository
+
+In case you want to use a different server/repository for a specific artifact to further customize your deployment, you can override the default URL in two ways:
+
+You can change the value of `component.repository` key for the selected component, provided that the path to your custom artifact follows the convention of the default `artifacts.alfresco.com` repository. For example to change the repository of ACS artifact you would:
```

Comment:
that convention is really a standard maven repository layout, we can better reference as that